### PR TITLE
Branch-2

### PR DESCRIPTION
**Pull request by Nekotekina**

### CODE REVIEW

File: rpcs3/Emu/Cell/SPURecompiler.cpp [hunk 1]
```diff
@@ -842,7 +842,7 @@ class spu_llvm_runtime
 	std::array<atomic_t<spu_function_t>, 0x10000> m_dispatcher;
 
 	// JIT instance (TODO: use small code model)
```
Comment (VelocityRa) [inline rpcs3/Emu/Cell/SPURecompiler.cpp]: Could this TODO be removed then?

**Comment by Rpcs3tester** [comment]
thanks @Nekotekina 

this fix The House Of Dead 4 [NPUB30603] 

previously it had a black screen before game play  

60 FPS but it's kind of slow , stutting .. this maybe same situation to Harry Potter And The Order Of The Phoenix Start screen only .

**Comment by legend800** [comment]
This PR broke GTI Club. Now it can no longer boot and throws an SPUASMJIT error immediately. Worked fine on the previous build. 

[RPCS3-GTI Neko.log.gz](https://github.com/RPCS3/rpcs3/files/1998189/RPCS3-GTI.Neko.log.gz)

Call stack:
=========
0x00000000 (0x0) called

·F 0:00:02.841621 {PPU[0x1000000] Thread (main_thread)} class std::runtime_error thrown: Verification failed (e=0xb7) (0): 
(in file Emu\Cell\SPUASMJITRecompiler.cpp:2753)
·! 0:00:02.841630 {PPU[0x1000000] Thread (main_thread)} Thread time: 0.031250s (0.153265Gc); Faults: 0 [rsx:0, spu:0];
·! 0:00:08.016791 Stopping emulator...
·! 0:00:08.016865 All threads signaled...
·! 0:00:08.016926 {Network Thread} Thread time: 0.000000s (0.107986Gc); Faults: 0 [rsx:0, spu:0];
·! 0:00:08.031925 All threads stopped...
·! 0:00:08.068372 Objects cleared...

**Comment by stevphie** [comment]
linux without LLVM ..Tales of Graces fall to 2-3 FPS.. used to be 45-60 fps..

when will this option enabled again  ?  and also wrong ICC profile for libpng 

tog-lowfps

**Comment by Parthorisian** [comment]
**SH: Downpour**[BLUS30565] worki for now https://i.imgur.com/BVieZYE.jpg
Log [downpourWorkinRPCS3.zip](https://github.com/RPCS3/rpcs3/files/2003198/downpourWorkinRPCS3.zip)

**Pull request merged**